hello_world_on_chain/lib/TAParserLib.sol

```diff
@@ -106,4 +116,13 @@ library TAParserLib {
 
         return claims;
     }
+
+    function base64d(
```

Comment:
updated some of the order of the functions to conform the the solidity style guide.